contributing.md

```diff
@@ -0,0 +1,68 @@
+* [Become a GitHub Contributor](#become-a-github-contributor)
+* [Approval Process](#approval-process)
+
+
+
+## Ways to Contribute
+
+There are several ways to contribute to LMHT depending on your level of github experience and what you want to accomplish:
+    * [Submit an Issue](#submit-an-issue)
+    * [Request a Feature](#request-a-feature)
+    * [Edit Documentation](#edit-documentation)
+    * [Become a GitHub Contributor](#become-a-github-contributor)
+
+
+## Submit an Issue
+
+Before submitting an issue, please visit the LMHT repository's [Issues page] to verify it is not already a known issue.  If is is a new issue, click "New Issue" in the upper right corner and title it according to what the issue is.  Issues are tracked by title so make it as accurate as possible.  Then leave a detailed comment about what the issue is, where it occurs, what button or feature were you trying to use, what did not work correctly, how should it have worked.  If possible submit screen shots, or any files that apply to document the issue.  Please do not assign any milestones or labels - we will do that.
+
+Once an issue is submitted, we will review it and request any additional information needed to verify the issue.  If it is a valid issue, we will keep it on the issues list to be addressed in the future.
+
+[Issues page]: https://github.com/LMHT/LMHT.github.io/issues
+
+## Request a Feature
+
+Currently, we are using the Issues page to also make feature requests.  Before making a feature request, please review the website and make sure that this feature is in line with the direction the site is going and is a new request.  It is up to you to submit a strong case for getting this feature approved.  If so, then follow the directions above for an issue, and submit your feature request.  Be as detailed as possible about what you want this feature to do.  We will review your case and decide accordingly.  Not all requests will be approved.
+
+
+## Edit Documentation
+
+There are two ways you can submit a change to documentation.  The [Issues page] can be used as well as [email].  Either way, make a copy of the documentation, edit it-clearly marking the changes you are requesting-and submit it.  Both email and the Issues page allow you to copy the document into your submission.
+
+[Issues page]: https://github.com/LMHT/LMHT.github.io/issues
+[email]: mailto:lmht@adorable.io
+
+## Become a GitHub Contributor
+
+If you wish to become a contributor to LMHT, please send an email to lmht@adorable.io that will include basic contact information, what areas you wish to contribute to (areas of expertise) such as english to spanish translation, coding, level of comfort using github.com, etc so that we can get your access set up.  As a contributor, the expectation is that you can use github fluently enough to fork branches, send pull requests and basically maneuver through github to send changes to update or correct the lmht pages as needed.
```

Comment:
Good point @LipGlossary. This document (Contributing.md) will only be visible on Github, in the repo. So people who see it already know how to use Github. The email system is meant for users who do not have experience on Github, and they have questions or a desire to contribute. So the Welcome Section on our live site is the appropriate place to mention details about the email. It won't hurt to mention our email address on this document or the Readme, though.